geomstats/geometry/poincare_polydisk.py

```diff
@@ -1,4 +1,18 @@
-"""The Poincare Polydisk."""
+"""
+The Poincare polydisk.
+
+The Poincare polydisk is defined as a product manifold of the Hyperbolic space
+of dimension 2. The Poincare polydisk has a product metric. The metric on each
+space is the natural Poincare metric multiplied by a constant.
+
+References
+----------
+The Kahler mean of Block-Toeplitz matrices
```

Comment:
Check out this example for references syntax.
Instead of [1], you can put a small alphanumeric id, e.g [JVKM2016]. Of course, you can keep the link at the end.

![image](https://user-images.githubusercontent.com/11378439/73088040-7b72cf80-3ea1-11ea-9d89-ae22b08cee93.png)